src/components/custom-hooks.jsx

```diff
@@ -0,0 +1,41 @@
+import {useState, useEffect}from 'react';
+
+const useFetch = (url, trigger = true) => {
```

Comment:
"It will be a logger, which logs api calls and save to a local storage"

Should also log to console, could be a future change